### ISSUE TITLE
Caught unhandled unknown exception, terminating

### ISSUE THREAD

**Issue by VortexLife** [davidsantos-br/ETS2Sync-Helper-4#33]
I keep getting this error whenever i try to sync jobs, i'm guessing this is needing updated?

**Comment by DaviMedrade** [comment]
Probably a duplicate of #31. If this still happens after #31 is closed, please reopen.
